### ISSUE TITLE
Is there any bug in MTCNN?

### ISSUE THREAD

**Issue by jixiaw** [TreB1eN/InsightFace_Pytorch#124]
In the forward function of class PNet: "a = F.softmax(a, dim=-1)",  why the dim is -1? Isn't it 1.
